app/assets/javascripts/application.js

```diff
@@ -11,8 +11,8 @@
 // about supported directives.
 //
 //= require jquery
-//= require jquery_ujs
-//= require popper
+//= require jquery-ujs
+//= require popper.js/dist/umd/popper.min.js
```

Comment:
```suggestion
//= require popper.js/
```

I can't really remember why I set `round-three` to require this distfile explicitly way back when I did that. I think we can require the package directly (perhaps back then I didn't even think to add the trailing `/`?).

The package's main entry points to `dist/umd/popper.js`, and there's no need to grab the minified version anyways since there's the double "sprockets will minify for us, and also minification isn't really needed in this day and age" factor.